Update OECD DAC Codelists with 2026-04-02 changes

Bjwebb opened the pull request:
The corresponding PR for our import scripts is https://github.com/IATI/DAC-Codelists/pull/14

This uses the new OECD development-finance-codelists: https://development-finance-codelists.oecd.org/CodesList.aspx

Bjwebb committed:
Update OECD DAC Codelists with 2026-04-02 changes

The corresponding PR for our import scripts is
https://github.com/IATI/DAC-Codelists/pull/14

This uses the new OECD development-finance-codelists:
https://development-finance-codelists.oecd.org/CodesList.aspx

markbrough commented:
I can see that there are a number of pending changes to the IATI codelists to bring them in line with the OECD codelists, and noticed this pull request. The Aid Transparency Tracker uses `iatikit`, which in turn uses the codelists from the IATI website. From [the IATI Dashboard](https://dashboard.iatistandard.org/exploring-data/codelists/2/default-aid-type@vocabulary1ornot@vocabulary_@code/), I can see that Australia, Denmark and Sweden are using the code B05, which is on the OECD codelists but not yet merged. Do you have an idea of when this PR could be merged by? Many thanks!

robredpath commented:
@markbrough I expect we'll get it out in the Gazette next week with a 30 day proposal period. Assuming no objections, we should see it merged by the end of May.

Bjwebb committed:
Add crs and tossd elements, to describe which other systems a code is in

These are namespaced, to avoid having to alter the standard versioned
IATI-Codelists xsd.

The namespace used is `extras` with the uri https://namespaces.iatistandard.org/codelist_extras

Bjwebb committed:
Add documentation of the `extras` namespace

Bjwebb committed:
Update OECD DAC Codelists with 2026-06-04 changes

The corresponding PR for our import scripts is
https://github.com/IATI/DAC-Codelists/pull/14

This uses the new OECD development-finance-codelists:
https://development-finance-codelists.oecd.org/CodesList.aspx

stevieflow reviewed (approved):
No objections received
